edb/server/protocol/auth_ext/_static/styles.css

```diff
@@ -232,25 +232,26 @@ a.field-note:hover {
 @media (prefers-color-scheme: dark) {
   body {
     background: #191C1F;
+    color: #DEE2E6;
```

Comment:
This is a change due to the default color of the UI text being the light-themed color still. Lemme know if there is a more appropriate body text color.